apps/backend/src/datasources/postgres/FileDataSource.ts

```diff
@@ -133,6 +138,84 @@ export default class PostgresFileDataSource implements FileDataSource {
       });
   }
 
+  private async storeBlobFromStream(
+    readableStream: NodeJS.ReadableStream
+  ): Promise<{ oid: number; sizeInBytes: number }> {

```

Comment:
Was there a suggestion to move this part to the factory to allow handling of  multiple proposals download.